### DISCUSSION TITLE
Can we track page visitors without using external trackers?

### DISCUSSION THREAD

**Discussion by jmaassen** [research-software-directory/RSD-as-a-service#181] [Q&A]
The question came up in the DCC meeting if it would be possible to track the number of page visitors for a software page, preferably without using external services such as google analytics (which may no longer be allowed in the EU due to GDPR issues). We could then show this as a metric on the software page (or private metrics page for this software if needed).

**Comment by ewan-escience** [comment]
How would we define a page visitor? Does each refresh count? Or will we track IP addresses or will we fingerprint visitors in a different way? 

We can create a solution ourselves or look at self hosted Google Analytics alternatives like [Plausible](https://plausible.io/docs/self-hosting) or [Matomo](https://matomo.org/pricing/). More alternatives [here](https://medevel.com/open-source-google-analytics-alternatives-2/).

**Reply by ctwhome** [reply]
Plausible.io is an European provider that doesn't requires to accept cockies because it complies fully with GDPR.
It could be a very interesting option to look at

From their website. 
"No cookies and fully compliant with GDPR, CCPA and PECR. Made and hosted in the EU, powered by European-owned cloud infrastructure 🇪🇺"

Some recognized front end developers are talking very well about this serviceb they describe as the Google analitics without the complexity. And it is also Open Source.
